### ISSUE TITLE
Black screen when phone is hot*

### ISSUE THREAD

**Issue by guillaumeboehm** [OlauncherCF/OlauncherCF#164]
#### Short description 
It seems that when my phone gets hot OlauncherCF just serves a black screen when going back to home, and gestures don't work. Switching to any other launcher isn't an issue, but even when switching back to OlauncherCF it still is just a black screen. Persists even some time after the phone cooled down, I have to force stop the app for it to work again.
Also Olauncher does not seem to have this issue.

#### Steps to reproduce 
I'm not even completely sure what's causing this but it always happenned when:
1. Drive in my car for a while with phone in the sun
2. At some point when phone is hot -> go to home screen
3. Lags a couple seconds before closing the app / unlocking phone
4. Black screen

#### Environment information 
- Olauncher CF Version: 1.1.0
- Install Source: F-Droid
- Device: Glaxy S10e
- Android: 12
- Other: optimization enabled 

#### Additional elements

If it helps the lags before the black screens happen only when the screen is "switched?". So if I unlock my phone with my fingerprint directly without waking it first it goes to the black screen without lagging.

Haven't tested with battery unrestricted yet. I unchecked it as I write this issue and I'll confirm if it happens again.

Very informative black screen:
![photo_2022-08-26_17-51-06](https://user-images.githubusercontent.com/12833794/186944326-0db084a2-295a-4a8a-9999-4ee2410271b0.jpg)

**Comment by jooooscha** [comment]
That's odd 😅

How many apps do you have on your home screen?
Can you reproduce this with zero apps on the home screen?

**Comment by jooooscha** [comment]
Sure, no problem. Take your time. Please report either way; when it works or not

**Comment by guillaumeboehm** [comment]
Ok I finally figured it out. There was indeed a correlation with heat but it wasn't the cause.
Turns out I used Google maps on long trips where my phone heats up a lot on its stand (coincidentally found this issue when I was starting to switch to osmand 😅 didn't make testing quicker lol) and the issue is actually when maps goes in picture in picture mode, the issue doesn't fix itself when maps is closed, only by force stopping olauncher. Although it's not very consistant because I managed to make it work without bugging...
I tested with other apps using picture in picture like twitch but the issue didn't occur.
I am still going to test it to try to nail the reproducibility.
